packaging/python/Dockerfile

```diff
@@ -11,24 +11,19 @@ RUN yum -y install \
     unzip \
     bison \
     flex \
-    pkg-config \
     autoconf \
     automake \
-    make \
     openssh-server \
-    libtool
+    libtool \
+    cmake3
 
 WORKDIR /root
 
-RUN curl -L -o cmake-3.12.3.tar.gz https://github.com/Kitware/CMake/releases/download/v3.12.3/cmake-3.12.3.tar.gz \
-    && tar -xvzf cmake-3.12.3.tar.gz \
-    && cd cmake-3.12.3 \
-    && ./bootstrap -- -DCMAKE_INSTALL_PREFIX=/nrnwheel/cmake \
-    && make -j install
+RUN ln -sf /usr/bin/cmake3 /usr/bin/cmake
 
-RUN wget http://ftpmirror.gnu.org/ncurses/ncurses-5.9.tar.gz \
-    && tar -xvzf ncurses-5.9.tar.gz \
-    && cd ncurses-5.9  \
+RUN wget http://ftpmirror.gnu.org/ncurses/ncurses-6.2.tar.gz \
```

Comment:
`libncurses` not packaged yet?

Reply:
@ferdonline there is the `.so`: 
```bash
[root@543c4a8a3040 ~]# rpm -qf /lib64/libncurses.so.5
ncurses-libs-5.9-14.20130511.el7_4.x86_64
```

However I thought we wanted it to be statically linked together with readline?

Reply:
There's 
`ncurses-static                         x86_64                         5.9`
```
[root@bac01514eced ~]# rpm -ql ncurses-static
/usr/lib64/libncurses.a
/usr/lib64/libncurses_g.a
/usr/lib64/libncursesw.a
/usr/lib64/libncursesw_g.a
...
```

Reply:
Ah great. Then I will try to use that one.

Reply:
Not straightforward, because `ncurses-devel` is a dependency for `ncurses-static`
```
[root@81280d8134bf ~]# yum deplist ncurses-static
Loaded plugins: fastestmirror, ovl
Loading mirror speeds from cached hostfile
 * base: linuxsoft.cern.ch
 * centos-sclo-rh: linuxsoft.cern.ch
 * epel: linuxsoft.cern.ch
 * extras: linuxsoft.cern.ch
 * updates: linuxsoft.cern.ch
package: ncurses-static.i686 5.9-14.20130511.el7_4
  dependency: ncurses-devel(x86-32) = 5.9-14.20130511.el7_4
   provider: ncurses-devel.i686 5.9-14.20130511.el7_4
package: ncurses-static.x86_64 5.9-14.20130511.el7_4
  dependency: ncurses-devel(x86-64) = 5.9-14.20130511.el7_4
   provider: ncurses-devel.x86_64 5.9-14.20130511.el7_4
```
Also `ncurses-static` is not relocatable.

Reply:
Not sure I understand. Why relocatable is needed? Even if we need the original .a file shipped, we can copy, no?

Reply:
Indeed, working on it now.

Reply:
I don't remember all details now but ncurses has ncurses, ncursesw etc. IIRC, `CFLAGS="-fPIC"` is required. So if system one doesn't work, its not big deal to build it.

Reply:
Yeah, not going to spend more time on it.  When we build the wheel we need to have those system libraries available in CMake. Will leave it like this for now.